packages/shared/src/components/RichTextEditor/plugins/ButtonPlugin/components/FloatingButton/EditButtonModal/EditModal.tsx

```diff
@@ -0,0 +1,45 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import React from 'react';
+import { FloatingButton } from '../FloatingButton';
+import { useFloatingButtonUrlInput } from '../FloatingButtonUrlInput';
+import { useGuidelineDesignTokens } from '../../../../../../../hooks';
```

Comment:
There are 5 occurences of the hook being called = 5 api call per RTE on the page, please optimize it

Reply:
As the designTokens are passed into the rte, the ones there should be used, so there shouldn't be any need to call `useGuidelineDesignTokens()` within the plugins

Reply:
Fixed to use the RTE context, which has the design tokens already